### PR TITLE
README: Remove finished "TODO" items

### PR DESCRIPTION
**Pull request by MarijnS95**
All devices are supported now thanks to the effort of @pablomh!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,15 +10,13 @@ Currently the path to the compilers, ramdisks and kernels are hardcoded, but eas
...
 1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path).
-2. Call `./oot/oot.sh <device name>`
-   Keep in mind that not all devices have a script yet.
+2. Call `./oot/oot.sh <device name>` (use `-h` to see more options)
 3. Flash the images. For simplicity, you can also add the `-f` option to the command, that will flash and `fastboot reboot` the device after building.
 
 ## TODO:
-- Add remaining devices
 - Move shared definitions to platforms? This includes sharing part of the cmdline across everything.
+- Expand platforms passed to this script
```
Comment (pablomh) [inline README.md]: Add period at the end of the sentence?

Comment (MarijnS95) [reply README.md]: Usually use `;` for this. Should be cleaned up now, most other points were anyway duplicates and/or already resolved.

File: README.md [hunk 2]
```diff
@@ -9,16 +9,10 @@ Currently the path to the compilers, ramdisks and kernels are hardcoded, but eas
 
 ## Usage:
 
-1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path).
-2. Call `./oot/oot.sh <device name>`
-   Keep in mind that not all devices have a script yet.
+1. Clone directly into your AOSP tree (or make sure `ANDROID_BUILD_TOP` is set to the right path);
```
Comment (pablomh) [inline README.md]: Maybe mention that `ANDROID_BUILD_TOP` is a variable with `${ANDROID_BUILD_TOP}`?

Comment (MarijnS95) [reply README.md]: Done :)

**Review by pablomh — APPROVED** [review]

**Pull request merged**